src/lib/slack/index.ts

```diff
@@ -40,12 +41,27 @@ const findUserIdByEmail = async (email: string) => {
 };
 
 export const sendCoursePoster = async (course: Course) => {
-  if (!isProduction) return;
-  const message = createBlocksNewTraining(course);
   const channel = SLACK_NEW_TRAININGS_CHANNEL;
+  if (!isProduction()) return;
+  const channelExistsResult = await channelExists(channel);
+  if (!channelExistsResult) return;
+  const message = createBlocksNewTraining(course);
   await sendMessage(channel, message);
 };
 
+export const channelExists = async (channel: string) => {
```

Comment:
I don't think this needs to be an export function

Reply:
Leftovers from testing the function, now removed